website/content/v1.12/reference/cli.md

```diff
@@ -100,14 +102,15 @@ talosctl cgroups [flags]
 ### Options
 
 ```
-      --cluster string       Cluster to connect to if a proxy endpoint is used.
-      --context string       Context to be used in command
-  -e, --endpoints strings    override default endpoints in Talos configuration
-  -h, --help                 help for cgroups
-  -n, --nodes strings        target the specified nodes
-      --preset string        preset name (one of: [cpu cpuset io memory process swap])
-      --schema-file string   path to the columns schema file
-      --talosconfig string   The path to the Talos configuration file. Defaults to 'TALOSCONFIG' env variable if set, otherwise '$HOME/.talos/config' and '/var/run/secrets/talos.dev/config' in order.
+      --cluster string             Cluster to connect to if a proxy endpoint is used.
+      --context string             Context to be used in command
+  -e, --endpoints strings          override default endpoints in Talos configuration
+  -h, --help                       help for cgroups
+  -n, --nodes strings              target the specified nodes
+      --preset string              preset name (one of: [cpu cpuset io memory process swap])
+      --schema-file string         path to the columns schema file
+      --siderov1-keys-dir string   The path to the SideroV1 auth PGP keys directory. Defaults to 'SIDEROV1_KEYS_DIR' env variable if set, otherwise '$HOME/.talos/keys'. Only valid for Contexts that use SideroV1 auth.
+      --talosconfig string         The path to the Talos configuration file. Defaults to 'TALOSCONFIG' env variable if set, otherwise '$HOME/.talos/config' and '/var/run/secrets/talos.dev/config' in order.
```

Comment:
please copy this manually to v1.11 reference as it is no longer updated automatically